local-antora-playbook.yml

```diff
@@ -24,7 +24,7 @@ asciidoc:
   - '@redpanda-data/docs-extensions-and-macros/macros/helm-ref'
 antora:
   extensions:
-  - require: '../docs-extensions-and-macros/extensions/test.js'
+  #- require: '@redpanda-data/docs-extensions-and-macros/extensions/unpublish-pages'
```

Comment:
Uncomment this line when this PR is merged: https://github.com/redpanda-data/docs-extensions-and-macros/pull/40